CMakeLists.txt

```diff
@@ -247,11 +247,25 @@ else()
   endif()
 
   if(${USING_ASAN} OR ${USING_ASAN_INT})
+
+    if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
```

Comment:
-fsanitize-blacklist and -fsanitize=integer only seem to be available with Clang so I added some checks